<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ventana</class>
 <widget class="QMainWindow" name="Ventana">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ventana</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="ObjNombre">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Salir">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>480</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Total">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>340</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Total</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Ingresar">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>290</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Ingresar Empleado</string>
    </property>
   </widget>
   <widget class="QComboBox" name="claseEmpleado">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Empleado Operativo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Empleado de Oficina</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="ObjSalario">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Consultar">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>70</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Consultar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ObjCedula">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>340</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;-- Informes de Salarios--&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>zona</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ObjNumEmpleados">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>250</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Salario</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="Salida1">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>120</y>
      <width>381</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Num Empleados</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Limpiar">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>70</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpiar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Promedio">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>330</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Promedio</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Consulta">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>70</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="Salida2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>390</y>
      <width>221</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ObjZona">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Clase Empleado</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="Salida3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>390</y>
      <width>221</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Cedula</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Bienvenido a UVNOMINA&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
